src/components/FeatureWalkthrough/FeatureWalkthrough.tsx

```diff
@@ -0,0 +1,203 @@
+import { FunctionComponent, useEffect } from 'react'
+
+import classNames from 'classnames'
+import { startCase } from 'lodash'
+import Link from 'next/link'
+
+import { buttonStyle, buttonLocation } from '@data'
+
+import batchChangesMp4 from '../../pages/home/assets/animations/batch-changes.mp4'
+import batchChangesWebm from '../../pages/home/assets/animations/batch-changes.webm'
+import codeInsightsMp4 from '../../pages/home/assets/animations/code-insights.mp4'
+import codeInsightsWebm from '../../pages/home/assets/animations/code-insights.webm'
+import codeIntelMp4 from '../../pages/home/assets/animations/code-intel.mp4'
+import codeIntelWebm from '../../pages/home/assets/animations/code-intel.webm'
+import codeSearchMp4 from '../../pages/home/assets/animations/code-search.mp4'
+import codeSearchWebm from '../../pages/home/assets/animations/code-search.webm'
+
+import { IFeatures } from './featureCopy'
+import { IVideoElement } from './video'
```

Comment:
I would just keep the interfaces scoped to this component since we aren't importing them elsewhere.

```suggestion
interface Video {
    mp4: string
    webm: string
}

interface VideoElement {
    el: HTMLVideoElement | null
    paused: boolean
}

interface Features {
    productFeature: string
    title: string
    description: string | ReactFragment
    details: string[]
    ctaLink: string
    video: Video
}
```